src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js

```diff
@@ -19,7 +19,6 @@ import {
   DataGenerator,
 } from '@elastic/charts';
 
-import { OUI_CHARTS_THEME_LIGHT } from '../../../../src/themes/charts/themes';
 
```

Comment:
```suggestion
```

The linter is throwing an error at this

Reply:
@BSFishy  I need to add a space.

Reply:
It looks like the issue is that there are 2 empty lines there, which gets caught by our linter. Removing one of them should fix the issue

Reply:
@BSFishy Done!.